api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -36,6 +36,7 @@ enum AccessLogType {
   NotSet = 0;
   TcpUpstreamConnected = 1;
   TcpPeriodic = 2;
+  TcpConnectionStart = 14;
```

Comment:
Can we rename it to `TcpDownstreamConnectionStart`?

Currently, the existing `TcpConnectionEnd` is for upstream connection which is confusing. Better specify with Tcp connection.